arches/app/views/manifest_manager.py

```diff
@@ -112,11 +112,15 @@ def create_canvas(image_json, file_url, file_name, image_id):
         def add_canvases(manifest, canvases):
             manifest.manifest["sequences"][0]["canvases"] += canvases
 
+        def check_canvas_in_use(canvas_id):
+            canvas_ids_in_use = [annotation.canvas for annotation in models.VwAnnotation.objects.all()]
+            return canvas_id in canvas_ids_in_use
```

Comment:
We don't need all the rows from the table to check if a single row exists containing `canvas_id`, so we can:
```suggestion
            return models.VwAnnotation.objects.filter(canvas__contains=canvas_id).exists()
```

Reply:
> We don't need all the rows from the table to check if a single row exists containing `canvas_id`, so we can:

Absolutely! Thanks.